lutris/installer/commands.py

```diff
@@ -216,7 +216,7 @@ def insert_disc(self, data):
               "<i>%s</i>") % requires
         )
         if self.runner == "wine":
-            GLib.idle_add(self.parent.eject_button.show)
+            self.parent.eject_button_show()
```

Comment:
why did you remove that,

Reply:
I moved it into the installer window.
https://github.com/lutris/lutris/pull/3029/files#diff-09064015f83e9cec3fca291ce9ae5954R538-R539

Otherwise the silent installer needs an eject_button with all the GUI dependencies.
This way I only need to provide an empty function.